examples/nlp/language_modeling/conf/megatron_gpt_inference.yaml

```diff
@@ -35,4 +35,9 @@ share: False  # whether create a public URL
 username: test # user name for web client
 password: test2  # password for web client
 web_port: 9889 # the port number of the web server
-chat: False # use the chat interface
\ No newline at end of file
+chat: False # use the chat interface
+chatbot_config:
+  value: False   # whether to inject the value attributes
```

Comment:
Can you call this steerlm_config instead?

Reply:
the paper is in anonymity. don't want to mention the name of the method.